download_sample_data.py

```diff
@@ -283,7 +278,6 @@ def main():
 
     The resulting list of files is then sampled and stored locally in the
     directory 'data' using a commonly used directory structure.
-
     """
     cfg_file = Path(__file__).parent.parent / "config.yml"
```

Comment:
```suggestion
    cfg_file = Path(__file__).parent / "config.yml"
```

Reply:
And: facets_file = Path(__file__).parent / "esmvaltool_sample_data" / "datasets.yml"